### PR TITLE
Kafka Deployment Events

### PR DESCRIPTION
**Pull request by KFBI1706**
This PR solves issue #36 and Naiserator is now able to publish Protobuf messages to Kafka once when a deployment is initialised and also when a deployment has succeed. 

I've also added tests, a makefile function and helpful docs in the README. If one wants to test it out for themselves.

What this feature implements is a kafka package that handles the kafka client , config, producer and event channel.
A deployment package which is just the package generated from the event.proto file. 
And a generators package which generates these events for us.

We've put most of the functionality into the synchronize loop, where it sends one event on initialization and monitors the app before sending a subsequent event on successful deployment.

I and @ambientsound have been working on this issue over slack and in person, so there's no discussion here on Github. But the solution is also pretty much done as far as I can tell. So it's basically ready to be merged.

Feedback is still very welcome!

### CODE REVIEW

File: pkg/kafka/client.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+type Client struct {
+	RecvQ         chan sarama.ConsumerMessage
+	Producer      sarama.SyncProducer
+	ProducerTopic string
+	SignatureKey  string
+}
+
+func tlsConfig(t TLS) *tls.Config {
+	return &tls.Config{
+		InsecureSkipVerify: t.Insecure,
```
Comment (VegarM) [inline pkg/kafka/client.go]: må vi❓

**Comment by ambientsound** [comment]
Veldig kjekt når man tester. Det er skrudd av i prod 😊

File: pkg/kafka/client.go (hunk 1 above)
Comment (KFBI1706) [reply pkg/kafka/client.go]: Nope, we really don't. Shamelessly copied this from navikt/deployment. I've cleaned up things a bit in my latest [commit](https://github.com/nais/naiserator/pull/93/commits/4cb9d2a421ace8153ba6dd9bfc9e8bb07a235433).

**Comment by KFBI1706** [comment]
Ja, fant ut at det var en docker kafka container i bakgrunnen som kjørte hele tiden. Som fikk meg til å tro at jeg hadde sjekket om kafka var enabled alle steder som trengtes...

**Pull request merged**